web_hierarchy_list/static/src/hierarchy_list_breadcrumb.esm.js

```diff
@@ -0,0 +1,15 @@
+import {Component} from "@odoo/owl";
+import {HierarchyListBreadcrumbItem} from "./hierarchy_list_breadcrumb_item.esm";
+
+export class HierarchyListBreadcrumb extends Component {
+    static components = {
+        HierarchyListBreadcrumbItem,
+    };
+    static props = {
+        parentRecords: Array[Object],
```

Comment:
i am pretty sure `Array[Object]` is just undefined. You probably mean `{ type: Array, element: Object}` (too bad, Array[Object] would be cool)